examples/argocd/providers.tf

```diff
@@ -1,5 +1,11 @@
 provider "aws" {
   region = var.region
+  default_tags = {
```

Comment:
It is incorrect declaration of default_tags for my terraform version. Does it work for your terraform version?